### PR TITLE
Can not use int with PyArg_ParseTuple since python 3.10 (fix for #58)

### PR DESCRIPTION
**Pull request by Janhouse**
I have no idea what I'm doing (not a C guy), so please review this.
This compiles and works for me with Python 3.10. Tested with my TV, seems good. But C allows too much stuff, which I am not an expert of, so some segfaults could be hiding there. :shrug: 

More info on the issue: https://bugs.python.org/issue40943

### CODE REVIEW

**Review by nforro — COMMENTED** [review]
LGTM :+1:

File: device.cpp [hunk 1]
```diff
@@ -25,6 +25,7 @@
 // request the std format macros
 #define __STDC_FORMAT_MACROS
 
+#define PY_SSIZE_T_CLEAN
```
[the remark this answers is not in the document]
Comment (Janhouse) [reply device.cpp]: Yes, I have it there as well.

[the remark this answers is not in the document]
Comment (Janhouse) [reply device.cpp]: Fixed

[the remark this answers is not in the document]
Comment (nforro) [reply device.cpp]: Thanks. Could you squash the commits into one?

[the remark this answers is not in the document]
Comment (Janhouse) [reply device.cpp]: Done

**Review by nforro — APPROVED** [review]

**Comment by trainman419** [comment]
Thanks for fixing this!

**Comment by trainman419** [comment]
I've released this to PyPi as version 0.2.8

**Pull request merged**